src/openApi/openApiWrapper.ts

```diff
@@ -66,7 +77,7 @@ export default class OpenApiWrapper {
             return new ApiResponse(error.statusCode, {
               title: error.message,
               details: error.details,
-              errorId: requestLogger.invocationId,
+              errorId: requestLogger.invocationId, // TODO: Keep?
```

Comment:
No harm in keeping it. We will log it twice and setting `errorId` to `correlationId` is kinda quesitonable

Reply:
Someone searching by the `errorId` will immediately find the trace through all services :D (I'm looking forward to always add a log query filter to see it in a specific service only).

I would leave it there though, many users will copy the body, but almost no one will copy the headers when reporting an issue. So this gives us a better chance to find the logs.

Reply:
That's a good point, more === better! :D

Reply:
could have dropped the TODO from the code :P

Reply:
Oh I did! But didn't push.. good that you checked :)